adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -2135,6 +2166,45 @@ trait MultisampleGenomicRDD[T, U <: MultisampleGenomicRDD[T, U]] extends Genomic
   }
 }
 
+/**
+ * A trait describing a GenomicDataset that also supports the Spark SQL APIs.
+ */
+trait DatasetBoundGenomicDataset[T, U <: Product, V <: GenomicDataset[T, U, V]] extends GenomicDataset[T, U, V] {
+  val uTag: TypeTag[U]
```

Comment:
shouldn't need `val uTag: TypeTag[U]` as that comes in from `GenomicDataset[T, U, V]`